optimizely/helpers/sdk_settings.py

```diff
@@ -32,7 +32,8 @@ def __init__(
             odp_segment_manager: Optional[OdpSegmentManager] = None,
             odp_event_manager: Optional[OdpEventManager] = None,
             fetch_segments_timeout: Optional[int] = None,
-            odp_event_timeout: Optional[int] = None
+            odp_event_timeout: Optional[int] = None,
+            odp_flush_interval: Optional[int] = None
```

Comment:
How about these names instead?
```suggestion
            odp_event_request_timeout: Optional[int] = None,
            odp_event_flush_interval: Optional[int] = None
```

Reply:
done